nscplugin/src/main/scala-3/scala/scalanative/nscplugin/NirGenStat.scala

```diff
@@ -129,10 +129,12 @@ trait NirGenStat(using Context) {
       if !f.isOneOf(Method | Module) && f.isTerm
     do
       given nir.Position = f.span
-
       val isStatic = f.is(JavaStatic) || f.isScalaStatic
       val isExtern = f.isExtern
       val mutable = isStatic || f.is(Mutable)
+      if (isExtern & !mutable) {
```

Comment:
Same as in the Scala 2 file, & -> &&

Reply:
Also it seems to be the root of problems in Scala 3 tests, after replacing & with && tests finished completely when I've tried it locally